tests/test_config/test_config.py

```diff
@@ -556,6 +562,23 @@ def test_multi_line_copyright(source_date_year, app, monkeypatch):
         ) in content
 
 
+@pytest.mark.parametrize('copyright_line', [
+    # '1970',
```

Comment:
Note: this test case is commented out.

Personally I think copyright notices that contain an individual year value should use the first-date-of-publication, because that's what's relevant when determining whether the content is out-of-copyright in various jurisdictions later-in-time.

If the individual year is instead the current year, then the observer has to do the same amount of work that they would if no copyright notice was present in order to figure out when the work was first published.

(we currently replace an individual value with the current year; I'm not sure that's ideal, but it isn't the purpose of this pull request, or the linked bugreport, to change that)

Reply:
I suppose the counterargument is that it's the _original_ work that has gone out of copyright when the term expires in each jurisdiction, and not the current/updated work.  But I don't know for certain whether that is the case.

Reply:
I think that https://www.newmediarights.org/book/can_same_book_have_multiple_copyright_dates provides some useful guidance about this; in any case, it's off-topic for this pull request but I wanted to highlight the ambiguity I find in this test case and why I commented it out.

Reply:
> highlight the ambiguity I find in this test case

To try to be pedantically clear: it seems that the ambiguity is at its utmost when no modification of the underlying work has occurred, and then a potentially-many-years-later rebuild of the documentation occurs.  In situations like that it feels particularly ungreat to me that rewriting of an existing year value could occur.

Reply:
In sensible build environments where `SOURCE_DATE_EPOCH` is determined by the last-modification-time of the work being built, then I suppose this is less of a problem.

Perhaps what I'm hand-wringing about is the fact that it could obscure the fact that many websites and/or software based works that display a single copyright year may in fact contain extensive work with legitimate copyright protection but that pre-dates the notice by a significant period of time, and thus may be available for use (something that an everyday person might not immediately understand).